_posts/2020/2020-02-02-calculate-a-mean-average-from-a-ruby-array.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Calculate a mean average from a Ruby array"
+description: "A lesson in why we use native Ruby methods"
+layout: article
+category: ruby
+image:
+  base: '2020/calculate-a-mean-average-from-a-ruby-array'
+  alt: "Black and White Texas Instruments calculator"
+  credit:  Ray Reyes
+  source: "https://unsplash.com/photos/3xwrg7Vv6Ts"
+
+---
+
+Ruby doesn’t provide a native method to generate an average (mean) value from an array of integers. Its built-in [math](https://ruby-doc.org/core-2.7.0/Math.html) library focuses on more complex calculations and there's no built-in `#average` or `#mean` method on `Array`.
```

Comment:
Well, 'maths' then, surely? As we ~Europeans~ Brits don't say 'math'? (But this is also not a big deal...)

Reply:
```suggestion
Ruby doesn’t provide a native method to generate an average (mean) value from an array of integers. Its built-in [Math](https://ruby-doc.org/core-2.7.0/Math.html) library focuses on more complex calculations and there's no built-in `#average` or `#mean` method on `Array`.
```